pkg/vmimage/data/validate.sh

```diff
@@ -37,3 +38,10 @@ oscap xccdf eval \
   /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml > /tmp/oscap.log || true
   
 python -c 'import sys; from lxml import html; h = html.fromstring(open("/tmp/scap-report.html").read()); sys.exit(1) if len(h.find_class("rule-result rule-result-fail")) > 0 else sys.exit(0)'
+
+SECURITY_LIST_SIZE=$(du /tmp/yum_updateinfo_list_security | cut -f1)
+if [ "$SECURITY_LIST_SIZE" -ne 0 ]; then
```

Comment:
This `du` check seems a bit convoluted, why not use the [`-s` built-in](http://www.tldp.org/LDP/abs/html/fto.html)?

```suggestion
if [ -s "/tmp/yum_updateinfo_list_security" ]; then
```

Reply:
Thanks! I had no idea about `-s`. Will try tomorrow